jest.conf/visual.load-test-components.js

```diff
@@ -6,18 +6,20 @@ import KButtonWithDropdownTest from '~~/lib/buttons-and-links/__tests__/componen
 import KDropdownMenuTest from '~~/lib/KDropdownMenu/__tests__/components/KDropdownMenuTest.vue';
 import KCheckboxSlotTest from '~~/lib/KCheckbox/__tests__/components/KCheckboxSlotTest.vue';
 import KBreadcrumbs from '~~/lib/KBreadcrumbs/index.vue';
-import KImgTest from '~~/docs/examples/KImg/Base.vue';
-import KImgAspectRatioTest from '~~/docs/examples/KImg/AspectRatio.vue';
-import KImgContentOnTopTest from '~~/docs/examples/KImg/ContentOnTop.vue';
-import KImgPlaceholderTest from '~~/docs/examples/KImg/Placeholder.vue';
-import KImgFitXYTest from '~~/docs/examples/KImg/FitXY.vue';
-import KImgFitCenterTest from '~~/docs/examples/KImg/FitCenterInside.vue';
-import KImgFitContainTest from '~~/docs/examples/KImg/FitContain.vue';
+import KImgTest from '~~/examples/KImg/Base.vue';
+import KImgAspectRatioTest from '~~/examples/KImg/AspectRatio.vue';
```

Comment:
Once we migrate the KImg test suite, we will need to give instructions to remove these components too!